.rubocop.yml

```diff
@@ -19,6 +19,9 @@ Layout/AlignParameters:
 Metrics/LineLength:
   Max: 120
 
+Metrics/MethodLength:
+  Max: 15
```

Comment:
Depends how often we are hitting this limit but if it's infrequently we should probably stick with the default and opt-out on a per-method basis. I guess you hit it here where you are returning a multiline hash even though it's logically just one line of code.

Reply:
correct. I can turn it off just for that instance.